README.md

```diff
@@ -9,6 +9,19 @@
 
 This is the Android client for the Mentorship System.
 
+## Features
+
+* A member gets an option to be a Mentor, Mentee or Both.
+* A member can build/customise the app profile with username, bio, skills, interests, location, occupation, etc. anytime. 
+* A member can view a list of other members and search for a member on the Members Page
+* A member can either send mentorship requests to other members as a Mentor or Mentee according to their interests or reject mentorship requests from other members.
+* A member can track the number of **Pending Resquests**, **Accepted Requests**, **Rejected Requests**, and **Completed Requests** and view **Recent Achievements** on the Home Page.  
+* A member can view the details of pending, past, all the mentorship reqests in the Requests Page.
+* A member can create, update, or delete tasks in their current mentorship relation.
+* A member can send feedback to the Mentorship System such as reporting a bug, giving suggestions or other comments.
```

Comment:
I think it should be "feedback about Mentorship system"

Reply:
Here I meant sending feedback 'to' the mentorship system developers 'about' its bugs,features,etc.
Should I change it? @MariamKotob @foongminwong

Reply:
I know that's what is meant but I think the member is basically sending feedback to the admins, maintainers and developers about their system that's why I believe it makes sense to be "about".

Reply:
Ahh okay I changed it:) @MariamKotob